src/_pytest/mark/legacy.py

```diff
@@ -106,5 +106,5 @@ def matchkeyword(colitem, keywordexpr):
             )
     try:
         return eval(keywordexpr, {}, mapping)
-    except SyntaxError:
+    except:
```

Comment:
`except:` is probably *too* broad, it also catches `Ctrl-C` and such, which should propagate. I think `except Exception` is the most appropriate.

Reply:
Fixed.

Reply:
It's not fixed ATM -- maybe you forgot to push?

Reply:
yeah. my bad. fixed now.